src/plugins/card/_class.scss

```diff
@@ -53,6 +53,12 @@ $picnic-card-radius: $picnic-radius !default;
     padding: .6em .8em 0;
   }
 
+  hr {
+    border: none;
+    height: 1px;
+    background-color: #eee;
```

Comment:
Should inherit somehow from the color to allow changing it. Probably:

```scss
border-top: $picnic-card-border;
```

or if you want to keep it as `background-color`, another variable would need to be defined like:

```scss
$picnic-card-border-color: #eee !default;
```